dashboard/v1.1/src/App.tsx

```diff
@@ -1,8 +1,32 @@
 import React from 'react';
 import './App.css';
+import {
+  createMuiTheme,
+  ThemeProvider,
+  responsiveFontSizes
+} from '@material-ui/core/styles';
+import { grey } from '@material-ui/core/colors';
+
+let theme = createMuiTheme({
+  palette: {
+    // Provides you with all
+    // shades of whites
+    primary: grey
+    // secondary: <Color>,
+  },
+  typography: {
+    fontFamily: ['Lato', 'Raleway'].join(','),
+    fontSize: 12
+  }
+});
+theme = responsiveFontSizes(theme);
 
 function App() {
-  return <div className="App">Bench-routes ui</div>;
+  return (
+    <ThemeProvider theme={theme}>
+      <div className="App">Bench-routes ui</div>
```

Comment:
can we split this into multiple lies? would appeal good.

Reply:
Can't because `prettier-fix` automatically reverts it back.